src/lib/utils/awell/slateToEscapedJsString/slateToEscapedJsString.ts

```diff
@@ -0,0 +1,70 @@
+import { JSDOM } from 'jsdom'
+
+export const slateToEscapedJsString = (html: string): string => {
+  // Use JSDOM to parse the HTML since we're in a Node.js environment
+  const dom = new JSDOM(html)
```

Comment:
I assume we can't use a tool to do this for us.

Reply:
Any suggestions you know of?